### PR TITLE
Devel coverage

### PR DESCRIPTION
**Pull request by bitbarbie**
I build a framework to include more mapping functions. Therefor the mapMetToGSMN function was moved to a single file, and it was plitted into two functions, the general call and the specific function for the code that was already in there (metabolites2network). On my system the vignette runs as before, as the default value fot he method chosen for the mapping in metabolites2network. 

Best case would be if @elvanov could merge to secure that the changes do not interfere with her recent changes.

### CODE REVIEW

File: R/mapMetToGSMN.R [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+#'
+#' @author Sarah Scharfenberg
+#'
+#' @export
+mapMetToGSMN <- function(inputData, method="metabolomics2network", resFile = "Res_Met2Net_MappedMet.txt", ...) {
+
+  if(method=="metabolomics2network"){
+
+    ## call function
+    mapMetToGSMN_metabolomics2network(inputData, resFile)
```
Comment (sneumann) [inline R/mapMetToGSMN.R:33]: Ah, so this is the original `mapMetToGSMN` Elva wrote. We now have this "new" `mapMetToGSMN()` as wrapper with the `method="metabolomics2network"` parameter to call the actual methods, just like xcms would select its peak picking functions like `centWave` or `matchedFilter`.

File: vignettes/gsmn_to_ontology.Rmd
Comment (sneumann) [inline vignettes/gsmn_to_ontology.Rmd:1]: is this a leftover ?

File: R/.gitignore
[the remark this answers is not in the document]
Comment (bitbarbie) [reply R/.gitignore:1]: Automatically created within RStudio when I add a file per right click -> git ignore.

File: R/mapMetToGSMN.R [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+#' `list`, list returned by the `loadInputData` function
+#'
+#' @param method one of Metabolomics2Network, molecularFormula, accurateMass, class_chebi or class_chemont
+#'
+#' @param ... additional parameter depending on the chosen function
+#'
+#' @return
+#' Data frame with the mappings and ontology-based distances.
+#'
+#' @author Sarah Scharfenberg
```
[the remark this answers is not in the document]
Comment (bitbarbie) [reply R/mapMetToGSMN.R:25]: Elva is assigned as author to the metabolites2networks individual function as solo author. I can add her here too of course.

File: R/mapMetToGSMN.R [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+  #   ## call function
+  #     ## one per chebi and chemont??
+  #     ## update this in documentation above
+  #   mapMetToGSMN_class()
+  # }
+}
+
+
+###################################################
+#      Singel functions not exported              #
```
[the remark this answers is not in the document]
Comment (bitbarbie) [reply R/mapMetToGSMN.R:67]: Thanks for the hint. I will add the . for the correct name conventions.

File: .gitignore [hunk 4]
```diff
@@ -3,3 +3,6 @@
 .RData
 .Ruserdata
 .Rproj
+MetClassNetR.Rproj
+20210804_classyfireR.db
```
[the remark this answers is not in the document]
Comment (bitbarbie) [reply .gitignore:7]: In my branch this is added in gitignore

**Comment by sneumann** [comment]
Hi, 
there is still a glitch in 
https://github.com/MetClassNet/MetClassNetR/blob/65052bc53074619d669196622e66fc9b74ac8ff1/R/MultiLayerFunctions.R#L532
see the output of our now working CI at:
https://github.com/MetClassNet/MetClassNetR/actions/runs/4925820385/jobs/8800483449#step:20:138
Yours, Steffen

**Comment by sneumann** [comment]
Merging, and we should have a joint code-base to continue from. Yours, Steffen

**Pull request merged**